pkg/services/ngalert/api/api_prometheus.go

```diff
@@ -222,7 +203,10 @@ func (srv PrometheusSrv) RouteGetRuleStatuses(c *models.ReqContext) response.Res
 		alertingRule.Rule = newRule
 		newGroup.Rules = append(newGroup.Rules, alertingRule)
 		newGroup.Interval = float64(rule.IntervalSeconds)
+		newGroup.EvaluationTime = newRule.EvaluationTime
```

Comment:
Do you want to take the earliest, latest, or just any value for EvaluationTime and LastEvaluation in the group? This will get modified on each iteration of the loop ...

Reply:
Yes, I mentioned this in the PR description. Grafana UI as well as alerting scheduler do not support alert grouping, and therefore, there is almost always only one rule in a group